dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -158,6 +158,18 @@ public void testDashboard()
       // grpc-testing-1.17.1, shown as first item in linkage errors, has these errors
       Assert.assertTrue(linkage.getValue().contains("3 linkage errors in 3 classes"));
 
+      ImmutableList<Node> dependencyPaths = toList(
+          document.query("//p[@class='static-linkage-check-dependency-paths']"));
+      Node log4jDependencyPathMessage = dependencyPaths.get(dependencyPaths.size() - 1);
+      // log4j dependency paths, which would have 994 lines, should be shown as summarized
```

Comment:
There are 994 paths to log4j. These should be summarized.

Reply:
Updated.